examples/step-6/step-6.cc

```diff
@@ -302,11 +299,14 @@ void Step6<dim>::assemble_system()
             {
               for (unsigned int j = 0; j < dofs_per_cell; ++j)
                 cell_matrix(i, j) +=
-                  (current_coefficient * fe_values.shape_grad(i, q_index) *
-                   fe_values.shape_grad(j, q_index) * fe_values.JxW(q_index));
-
-              cell_rhs(i) += (fe_values.shape_value(i, q_index) * 1.0 *
-                              fe_values.JxW(q_index));
+                  (current_coefficient *              // a(x_q)
+                   fe_values.shape_grad(i, q_index) * // grad phi_j(x_q)
+                   fe_values.shape_grad(j, q_index) * // grad phi_i(x_q)
```

Comment:
you got the indices in the comment wrong -- exchange the i and j here

Reply:
fixed!